tests/vmi_configuration_test.go

```diff
@@ -531,8 +533,10 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 			It("[test_id:5266]should boot to NIC rom if a boot order was set on a network interface", func() {
 				By("Creating a VMI with no disk and an explicit network interface")
-				vmi := tests.NewRandomVMI()
```

Comment:
`tests.NewRandomVMI()` seems to be VMI without disks. Is this correct?

Reply:
That's a good catch, thanks!